config/initializers/devise.rb

```diff
@@ -275,6 +275,8 @@
   # Add a new OmniAuth provider. Check the wiki for more information on setting
   # up on your models and hooks.
   # config.omniauth :github, 'APP_ID', 'APP_SECRET', scope: 'user,public_repo'
+  config.omniauth :bird, strategy_class: OmniAuth::Strategies::Bird if Settings.omniauth.bird.enable
+  config.omniauth :samltestid, strategy_class: OmniAuth::Strategies::SamlTestId if Settings.omniauth.samltestid.enable
```

Comment:
Disabling providers does not yet work as expected which results in the rspec failure and the links being displayed if the strategy should be disabled.

Reply:
This issue has been resolved.